glean-core/src/metrics/timing_distribution.rs

```diff
@@ -138,9 +138,10 @@ impl TimingDistributionMetric {
 
     /// Starts tracking time for the provided metric.
     ///
-    /// This records an error if it’s already tracking time (i.e. start was already
-    /// called with no corresponding [stop]): in that case the original
-    /// start time will be preserved.
+    /// This records an error if it’s already tracking time (i.e.
+    /// [TimingDistributionMetric::set_start] was already called with no
+    /// corresponding [TimingDistributionMetric::set_stop_and_accumulate]): in
```

Comment:
We could clean this up like this to make it shorter:

```suggestion
    /// [`set_start`](TimingDistributionMetric::set_start) was already called with no
    /// corresponding [`set_stop_and_accumulate`](TimingDistributionMetric::set_stop_and_accumulate)): in
```